frontend/src/app/shared/authentication.interceptor.ts

```diff
@@ -0,0 +1,25 @@
+import { inject } from '@angular/core';
+import { HttpInterceptorFn } from '@angular/common/http';
+import { first, switchMap } from 'rxjs';
+import { OidcSecurityService } from 'angular-auth-oidc-client';
+
+export const authenticationInterceptor: HttpInterceptorFn = (req, next) => {
+  // we can't depend directly on OidcSecurityService, because (as documented), it would introduce
+  // a dependency cycle
```

Comment:
But we do now. And apparently it works fine. So we can delete the comment above.